R/build-redirects.R

```diff
@@ -78,25 +62,15 @@ build_redirect <- function(entry, index, pkg) {
   write_lines(lines, old)
 }
 
-article_redirects <- function(pkg) {
-  if (is.null(pkg$meta$url)) {
-    return(NULL)
-  }
-
-  is_vig_in_articles <- path_has_parent(pkg$vignettes$name, "articles")
-  if (!any(is_vig_in_articles)) {
-    return(NULL)
-  }
-
-  articles <- pkg$vignettes$file_out[is_vig_in_articles]
-  purrr::map(articles, ~ paste0(c("articles/", ""), .x))
+data_redirects <- function(pkg, has_url = FALSE, call = caller_env()) {
```

Comment:
Introduced new `data_redirects`, like most of the other components, to make testing easier.